### ISSUE TITLE
CT alignBounds should be a non-empty finite Bounds2

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/balloons-and-static-electricity#565]
```
balloons-and-static-electricity : phet-io-state-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1655230883388/phet-io-wrappers/state/?sim=balloons-and-static-electricity&phetioDebug=true&phetioWrapperDebug=true&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22balloons-and-static-electricity%22%2C%22phet-io-state-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1655230883388%22%2C%22timestamp%22%3A1655233458274%7D
Uncaught Error: Uncaught Error: Assertion failed: alignBounds should be a non-empty finite Bounds2
Error: Assertion failed: alignBounds should be a non-empty finite Bounds2
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1655230883388/assert/js/assert.js:28:13)
at assert (AlignBox.ts:206:14)
at setAlignBounds (AlignBox.ts:225:50)
at (AlignGroup.ts:285:4)
at setBoxBounds (AlignGroup.ts:248:13)
at updateLayout (AlignGroup.ts:294:9)
at onAlignBoxResized (AlignBox.ts:189:18)
at listener (TinyEmitter.ts:94:8)
at emit (TinyStaticProperty.ts:58:9)
at notifyListeners (Node.ts:1486:30)
id: Bayes Puppeteer
Snapshot from 6/14/2022, 12:21:23 PM
```

**Comment by jessegreenberg** [comment]
I believe this is from https://github.com/phetsims/joist/issues/809#issuecomment-1164775473

**Comment by jessegreenberg** [comment]
Fixed through phetsims/joist#809
